src/zenml/integrations/kaniko/image_builders/kaniko_image_builder.py

```diff
@@ -376,14 +456,10 @@ def _check_prerequisites() -> None:
             RuntimeError: If any of the prerequisites are not installed.
         """
         if not shutil.which("kubectl"):
-            raise RuntimeError(
-                "`kubectl` is required to run the Kaniko image builder."
-            )
+            raise RuntimeError("`kubectl` is required to run the Kaniko image builder.")
```

Comment:
This check isn't accurate anymore. You should try testing your changes without `kubectl` installed.